### ISSUE TITLE
OKEX - Order History Response Not Parsed Properly - Amount incorrect

### ISSUE THREAD

**Issue by azharuniverse** [ccxt/ccxt#3510]
Amount of market buy order is coming zero but is available in raw order object. 
`{
  "info": {
    "amount": 0,
    "avg_price": 0.00045553,
    "create_date": 1532752598000,
    "deal_amount": 3.2862,
    "order_id": 407895054,
    "orders_id": 407895054,
    "price": 0.00149699,
    "status": 2,
    "symbol": "okb_btc",
    "type": "buy_market"
  },
  "id": "407895054",
  "timestamp": 1532752598000,
  "datetime": "2018-07-28T04:36:38.000Z",
  "symbol": "OKB/BTC",
  "type": "market",
  "side": "buy",
  "price": 0.00149699,
  "average": 0.00045553,
  "cost": 0.001496962686,
  "amount": 0,
  "filled": 3.2862,
  "remaining": 0,
  "status": "closed"
}`

Here info.deal_amount contains actual order amount. 

- OS: ALL
- Programming Language version: NODE v8.9.4
- CCXT version: 1.16.36
- Exchange: OKEX, OKCOINUSD
- Method: fetchClosedOrders

**Comment by kroitor** [comment]
```
{ info:
   { amount: 0,
     avg_price: 0.00045553,
     create_date: 1532752598000,
     deal_amount: 3.2862,  // ←---------------------------- this value
     order_id: 407895054,
     orders_id: 407895054,
     price: 0.00149699,
     status: 2,
     symbol: 'okb_btc',
     type: 'buy_market' },
  id: '407895054',
  timestamp: 1532752598000,
  datetime: '2018-07-28T04:36:38.000Z',
  symbol: 'OKB/BTC',
  type: 'market',
  side: 'buy',
  price: 0.00149699,
  average: 0.00045553,
  cost: 0.001496962686,
  amount: 0,
  filled: 3.2862,  // ←---------------------------- is here
  remaining: 0,
  status: 'closed' }
```

https://github.com/ccxt/ccxt/wiki/Manual#order-structure

Let us know if the above does not help.

**Comment by kroitor** [comment]
This is fixed in 1.17.37. Let us know if you have any difficulties with it. Thx for reporting!

**Comment by azharuniverse** [comment]
Its working correctly in 1.17.37
Thanks for quick reply. 
I am always pleased with CCXT work and support.
